src/network/networkd-dhcp4.c

```diff
@@ -1704,17 +1704,24 @@ int config_parse_dhcp_ip_service_type(
                 void *data,
                 void *userdata) {
 
+        int *tos = data;
+
         assert(filename);
         assert(lvalue);
         assert(rvalue);
+        assert(data);
```

Comment:
`assert(tos)` instead of `data`?

Reply:
No difference, but I usually use `assert(data)` or `assert(userdata)` in `conf_parsers()`.